liveCornerPlot.py

```diff
@@ -0,0 +1,217 @@
+from matplotlib import pyplot as plt
```

Comment:
I don't think this file should live in the main repo; it looks more like a useful but custom script for generating corner plots from a chain file for one object. The main repo should be more for general purpose stuff that would be used all the time.

A standalone corner plot script might make sense, but needs to be quite different from this. There are several things that might need changing for such a script:

1. You define a long ```corner``` function, which I think is then not used?
1. The columns are hard coded, which is inelegant
1. The template will only work for complex BS models.

A more elegant approach might be to use the input file used for the mcmc run to define a mapping between parameter name and column in the chain, and allow the user to specify somehow which parameters are desired in the chain...

Reply:
In hindsight it's definitely a custom script, doesn't belong here.